### PR TITLE
update test scenarios

### PR DESCRIPTION
**Pull request by nam-jaehyun**
- update test scenarios
- update example policies with how to test each policy

### CODE REVIEW

File: getting-started/security_policy_examples.md [hunk 1]
```diff
@@ -71,36 +71,38 @@ Here, we demonstrate how to define security policies using our example microserv
...
     kind: KubeArmorPolicy
     metadata:
-      name: ksp-ubuntu-3-proc-dir-allow
+      name: ksp-ubuntu-3-file-dir-allow-from-source-path
       namespace: multiubuntu
     spec:
+      severity: 10
+      message: "a critical directory was accessed"
+      tags:
+      - WARNNING
```
Comment (daemon1024) [inline getting-started/security_policy_examples.md]:
```suggestion
      - WARNING
```

**Review by daemon1024 — COMMENTED** [review]
Small nit, Rest LGTM 👌🏽 

Side note, What do you think about adding a CI to run multiubuntu test scenario on PRs, but since this is an extended test suit it should only run if manually triggered.

File: getting-started/security_policy_examples.md (hunk 1 above)
Comment (nam-jaehyun) [reply getting-started/security_policy_examples.md]: fixed the typo in 'security_policy_examples.md' and other files too.

**Comment by nam-jaehyun** [comment]
> Small nit, Rest LGTM 👌🏽
> 
> Side note, What do you think about adding a CI to run multiubuntu test scenario on PRs, but since this is an extended test suit it should only run if manually triggered.

The reason that we use github test scenarios instead of multiubuntu ones is that it takes too much time.

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**